gsd/test/test_fl.py

```diff
@@ -40,7 +40,7 @@ def test_create(tmp_path):
 ])
 def test_dtype(tmp_path, typ):
     """Test all supported data types."""
-    data1d = numpy.array([1, 2, 3, 4, 5, 10012], dtype=typ)
+    data1d = numpy.array([1, 2, 3, 4, 5, 127], dtype=typ)
```

Comment:
Why this change?

Reply:
This quiets a deprecation warning from numpy. `test_dtype` is called with the `int8` data type which cannot represent the value 10012.